ui/job-view/details/tabs/TabsPanel.jsx

```diff
@@ -189,8 +190,9 @@ class TabsPanel extends React.Component {
                 )}
                 <FontAwesomeIcon
                   icon={isPinBoardVisible ? faAngleDown : faAngleUp}
+                  className="ml-1"
                 />
-              </span>
+              </Button>
               <span
                 onClick={() => clearSelectedJob(countPinnedJobs)}
```

Comment:
This one probably needs aria-label="Close" role="button" or similar.

Reply:
Good call.  Done.